src/plugins/IORedisPlugin.ts

```diff
@@ -49,7 +49,7 @@ class IORedisPlugin implements SwPlugin {
       span.layer = SpanLayer.CACHE;
       span.peer = host;
       span.tag(Tag.dbType('Redis'));
-      span.tag(Tag.dbInstance(`${this.condition.select}`));
+      span.tag(Tag.dbInstance(`${this.condition?.select}`));
```

Comment:
Could the `condition` be null? Could you share in which cases?

Reply:
As described in https://github.com/apache/skywalking/issues/7275,  here is what happened in my app:
![image](https://user-images.githubusercontent.com/21286377/155684289-a39a65bb-276a-470e-8eb5-04fc7ffeda48.png)

Reply:
I got this. I mean is there a solution to make sure `dbInstance` would be set properly? Because this is an important tag, it is expected by the backend.

Reply:
As we use redis cluster, `dbInstance` makes no sense. Also, there's no `condition` property in redis cluster mode.

Reply:
For cluster mode, instance should be a cluster name or IP port list.
This is recommended.

Reply:
Redis cluster mode can be  inferred by span,  and here `dbInstance` actually means zero-based numeric index of Redis accounding to the repo.

![image](https://user-images.githubusercontent.com/21286377/155692943-5b0c38e5-3406-49b0-8d53-0aeb1ee67177.png)

Reply:
There is no cache dashboard to represent metrics for performance yet, but once it is added, this undefined name will show up in the cache server list. 
So, only an endpoint name is not enough. I can see peer includes an IP:port. Isn't it for the Redis server?

Reply:
The Peer here stands for one of Redis cluster nodes. Is this value suitable for `dbInstance `?

Reply:
I am expecting the whole `IP1:port1, IP2:port2,...`

Reply:
Redis cluster nodes cannot be acquired by the code contexts unless we call some function which I believe is not appropriate.

Reply:
Then at least use the peer value. It is at least better than null.